### DISCUSSION TITLE
TabBar 的 onChange 为什么在同 key 的情况也会触发

### DISCUSSION THREAD

**Discussion by a145789** [ant-design/ant-design-mobile#5408] [Q&A] [answered]
连续点击 TabBar 的某一 Item ,即使是 key 值相同，onChange 也会触发，onChange 事件应该是在 key 值不同的时候触发吧。

疑惑的点是，顺便看了下其他的 UI 库，除了 vant 其他的也都是相同逻辑，只有 vant 在 key 值相同时不触发。设计如此么？

**Comment by awmleer** [comment, marked as answer]
应该是个 bug，我开一条 issue

**Reply by awmleer** [reply]
update 一下，已经修复了，等待下个版本发布
